static/app/views/starfish/queries/useSpanMetrics.tsx

```diff
@@ -9,27 +10,35 @@ import {
   MetricsResponse,
   SpanMetricsQueryFilters,
 } from 'sentry/views/starfish/types';
-import {useSpansQuery} from 'sentry/views/starfish/utils/useSpansQuery';
+import {useWrappedDiscoverQuery} from 'sentry/views/starfish/utils/useSpansQuery';
 import {EMPTY_OPTION_VALUE} from 'sentry/views/starfish/views/spans/selectors/emptyOption';
 
 export const useSpanMetrics = <T extends MetricsProperty[]>(
   filters: SpanMetricsQueryFilters,
   fields: T,
-  referrer: string = 'span-metrics'
+  sorts?: Sort[],
+  limit?: number,
+  cursor?: string,
+  referrer: string = 'api.starfish.use-span-metrics'
```

Comment:
Also for future PR what do you think about changing this so that you always have to pass in a referrer. This should make it way easier for snuba to identify who exactly is the callee.

Should we also try to enumify the referrers so we don't collide names? We do this in backend https://github.com/getsentry/sentry/blob/2aa7d3ebd7885a77e99848712979fbbc087b9bb6/src/sentry/api/endpoints/organization_events.py#L31

Reply:
Yep, this tracks, I'll add a todo for this 👍🏻